docs/release.md

```diff
@@ -119,3 +119,23 @@ git push $YOUR_REMOTE release-0.6:release-0.6-aParticularFix
 After pushing the branch, you'll need to make sure to create a pull request
 against the correct target branch in GitHub (in this case the target branch
 is `release-0.6`).
+
+Releasing patch versions
+------------------------
+
+Releases on the stable branch only increment the patch level.
+The release itself is only a evsigned tag as it's used for minor releases as well.
+
+1. Tag the commit using `git evtag sign $TAG` (which is a signed and annotated
```

Comment:
Successful execution of this command appears to rely on two things.

* git-evtag is installed
* gpg is set up with at least one keypair that can be used to do the signing

Assuming that's true, will we want to have some formalized process of distributing public keys?

Reply:
Good idea, I'll link to
https://help.github.com/articles/adding-a-new-gpg-key-to-your-github-account/

Reply:
Done.